### PR TITLE
Update files for LANL IC machine chicoma-cpu

### PR DESCRIPTION
**Pull request by jonbob**
Updates machine files for chicoma-cpu after the modules were changed and new project space added

[BFB]

### CODE REVIEW

File: cime_config/machines/config_machines.xml [hunk 1]
```diff
@@ -3417,15 +3417,20 @@
...
 
       <modules compiler="nvidia">
-        <command name="load">PrgEnv-nvidia</command>
+        <command name="load">PrgEnv-nvidia/8.3.3</command>
         <command name="load">nvidia/22.3</command>
       </modules>
 
+      <modules compiler="intel">
+        <command name="load">PrgEnv-intel/8.3.3</command>
+        <command name="load">intel-classic/2022.2.1</command>
```
Comment (xylar) [inline cime_config/machines/config_machines.xml:3431]: @jonbob, have you been able to build E3SM successfully with this configuration?  If so, I'd like to add it to mache (and compass and polaris).

Comment (jonbob) [reply cime_config/machines/config_machines.xml:3431]: @xylar -- yes, and this is what I gave to Irena for her recent runs.

Comment (xylar) [reply cime_config/machines/config_machines.xml:3431]: Ah, okay.  I hadn't been paying attention.  I assumed she was building with `gnu` still.

Comment (jonbob) [reply cime_config/machines/config_machines.xml:3431]: Oh, my bad -- I wasn't paying attention to the specifics of your question. No, I still haven't gotten it to run with intel, but wanted to have the correct module for testing and further work

Comment (xylar) [reply cime_config/machines/config_machines.xml:3431]: In her build script, she isn't specifying a compiler so I think it's defaulting to `gnu` still.

Comment (xylar) [reply cime_config/machines/config_machines.xml:3431]: (Unless that got changed from the version of the build script I saw last.)

Comment (xylar) [reply cime_config/machines/config_machines.xml:3431]: > No, I still haven't gotten it to run with intel, but wanted to have the correct module for testing and further work.

Okay I'll hold off on this until I hear how things go.

**Review by darincomeau — APPROVED** [review]
Approved based on successfully running a smoke branch (using default gnu).

Thanks @jonbob !

**Comment by jonbob** [comment]
@xylar -- are you ok with this going in as-is?

**Comment by xylar** [comment]
Yes, I wasn't actually asked to review so I was just commenting for my own information.

**Review by xylar — APPROVED** [review]
Approving based on Irena Vankova's successful runs.

**Comment by jonbob** [comment]
passes sanity testing -- merged to next

**Comment by jonbob** [comment]
merged to master

**Pull request merged**